modules/images.js

```diff
@@ -0,0 +1,25 @@
+var Images = "";
+
+function randomIcons(){
```

Comment:
Use verb phrases for your functions. Currently, they are difficult to understand and this increases your code complexity by a lot.

Reply:
okay sir

Reply:
@devbash1 here too